tests/hardhat/Fork/ForceVAIDebtFirstTest.ts

```diff
@@ -76,6 +79,15 @@ async function deployAndConfigureLiquidator() {
   liquidatorNew = Liquidator__factory.connect(LIQUIDATOR, impersonatedTimelock);
 }
 
+async function deployAndConfigureComptroller() {
+  const comptrollerFactory = await ethers.getContractFactory("Comptroller");
+  const comptrollerImpl = await comptrollerFactory.deploy();
+  await comptrollerImpl.deployed();
+  const comptrollerProxy = Unitroller__factory.connect(UNITROLLER, impersonatedTimelock);
+  await comptrollerProxy.connect(impersonatedTimelock)._setPendingImplementation(comptrollerImpl.address);
+  await comptrollerImpl.connect(impersonatedTimelock)._become(UNITROLLER);
+}
+
```

Comment:
I would wait for executing [VIP-172](https://app.venus.io/#/governance/proposal/172) and then I would update the block number. This way we won't need to do this (or granting permissions on `_setForcedLiquidation`)

Reply:
ACK

Reply:
Fixed [bd621f8](https://github.com/VenusProtocol/venus-protocol/pull/241/commits/bd621f8bea9e39bcf06c7c5bc24a90dda5b0d2f5)